### PR TITLE
Mlflow logger

### PR DESCRIPTION
**Pull request by AnsuFrancis**
Usage:
```
import mlflow
from dkube.sdk.mlflow_logger import NotebookMlflowLogger
nblogger = NotebookMlflowLogger()
logger = nblogger.get_logger()


with mlflow.start_run(run_name="insurance") as run:
    logger.info("complete")
    nblogger.flush()
```

### CODE REVIEW

File: dkube/sdk/mlflow_logger.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            logger.addHandler(console_handler)
+            return logger
+
+    def flush(self):
+        logger = logging.getLogger()
+        log_paths = [handler.baseFilename for handler in logger.handlers if isinstance(
+            handler, logging.FileHandler)]
+        log_file = next(iter(log_paths))
+        if mlflow.active_run():
+            print("Persisting logs in storage")
```
Comment (mak-454) [inline dkube/sdk/mlflow_logger.py]: remove this

File: dkube/sdk/mlflow_logger.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import logging
+import sys
+import mlflow
+import tempfile
+import os
+
+
+class NotebookMlflowLogger:
```
Comment (mak-454) [inline dkube/sdk/mlflow_logger.py:8]: add check for JOB_CLASS = notebook

File: dkube/sdk/mlflow_logger.py (hunk 1 above)
Comment (AnsuFrancis) [reply dkube/sdk/mlflow_logger.py]: addressed

File: dkube/sdk/mlflow_logger.py (hunk 2 above)
Comment (AnsuFrancis) [reply dkube/sdk/mlflow_logger.py:8]: addressed

**Pull request merged**